jabgui/src/main/java/org/jabref/gui/theme/ThemeManager.java

```diff
@@ -73,7 +73,13 @@ public ThemeManager(WorkspacePreferences workspacePreferences,
         this.workspacePreferences = Objects.requireNonNull(workspacePreferences);
         this.fileUpdateMonitor = Objects.requireNonNull(fileUpdateMonitor);
         this.updateRunner = Objects.requireNonNull(updateRunner);
-        this.themeWindowManager = ThemeWindowManagerFactory.create();
+        ThemeWindowManager tempThemeWindowManager = null;
```

Comment:
We do not use "temp" prefix in JabRef. Do without. 

With "this." you have the different scoping.

Maybe do without local variable and in the catch block set to null?